src/hb-common.cc

```diff
@@ -1039,6 +1043,56 @@ hb_variation_from_string (const char *str, int len,
   return false;
 }
 
+#if !defined(HARFBUZZ_NO_SETLOCALE) && defined(HAVE_NEWLOCALE) && defined(HAVE_USELOCALE)
```

Comment:
Should this be `HB_NO_SETLOCALE`?

IIUC, if I set, `HB_NO_SETLOCALE`, it looks like I still need to include a `locale.h` or `xlocale.h` on Linux and Mac respectively in order to find a `locale_t` even though the calls are a no-op? Can that be avoided?

Reply:
Fixed. Thanks.